### ISSUE TITLE
Is it possible to restrict user operations?

### ISSUE THREAD

**Issue by QqQss** [lilab-bcb/cirrocumulus#209]
Hi guys, 

Thanks for developing such an amazing visualization framework!

The question I have right now is whether it's possible to limit what users can do in the "server" mode, so they can only browse the data pre-loaded in the webserver without being able to perform advanced operations like deleting or adding new datasets.

Thanks!

**Comment by joshua-gould** [comment]
Yes, you need to enable authentication using Google or Okta. Please see https://cirrocumulus.readthedocs.io/en/latest/installation.html#server-mode.
